model/atmosphere/diffusion/tests/diffusion_tests/mpi_tests/test_parallel_diffusion.py

```diff
@@ -168,7 +168,7 @@ def test_parallel_diffusion_multiple_steps(
     caplog,
     backend,
 ):
-    if settings.dace_orchestration is None:
+    if "dace" not in backend.name.lower():
         raise pytest.skip("This test is only executed for `--dace-orchestration=True`.")
```

Comment:
Could you please remove this option from the pytest configuration?
This is not needed anymore.
Thank you!